.env.example

```diff
@@ -0,0 +1 @@
+#NEXT_PUBLIC_INFURA_PROJECT_ID="xxx"
```

Comment:
`NEXT_PUBLIC` prefix is not needed here, right?

Reply:
Yeah, it's not needed. I'll remove it to avoid any confusion